java/mykola-melnyk/src/com/zoolatech/lecture7/tasks/_2/Calc_v_2.java

```diff
@@ -0,0 +1,33 @@
+package com.zoolatech.lecture7.tasks._2;
+
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+public class Calc_v_2 implements Calc {
+    private volatile int storedValue;
```

Comment:
same as in a previous version:
1. volatile can be removed
2. getter should also use the lock object

Reply:
Done